templates/components/textfield.html

```diff
@@ -0,0 +1,31 @@
+<!-- prettier-ignore -->
+{% let input_type %}
+{% if name == "password" || name == "confirm-password" %}
+    {% let input_type = "password" %}
+{% else %}
+    {% let input_type = "text" %}
+{% endif %}
+
+{% let border_color %}
+{% if error_message == "" %}
+    {% let border_color = "border-gray-300" %}
+{% else %}
+    {% let border_color = "border-red-500" %}
+{% endif %}
```

Comment:
Přesně takhle jsem cca myslel, že bychom udělali tu theme

Reply:
Mám pocit, že ty proměnné jsou ale validní jen v rámci jednoho templatu, takže to nejspíš nepůjde, ale nekontroloval jsem to.